firewall-examples.md

```diff
@@ -4,9 +4,7 @@ Stability: 4
 ----
 Text:
 
-				    <div class='panelMacro'><table class='infoMacro'><colgroup><col width='24'><col></colgroup><tr><td valign='top'><img src="/assets/images/icons/emoticons/information.gif" width="16" height="16" align="absmiddle" alt="" border="0"></td><td><b>Beta</b><br />Cloud Firewall is in beta at this time.</td></tr></table></div>
-
-<p>These examples are meant to outline some common use-cases for the firewall. Note that all instances need to have their firewall property set for rules to apply to them:</p>
+These examples are meant to outline some common use-cases for the firewall. Note that all instances need to have their firewall property set for rules to apply to them:
 
 <div class="code panel" style="border-width: 1px;"><div class="codeContent panelContent">
 <pre class="theme: Default; brush: plain; gutter: false" style="font-size:12px; font-family: ConfluenceInstalledFont,monospace;">
```

Comment:
I'd format this as follows:

```bash
$ sdc-enablemachinefirewall <machine-id>
```

```
```bash
$ sdc-enablemachinefirewall <machine-id>
```
```